src/config/auth.config.ts

```diff
@@ -150,6 +150,7 @@ export default {
     signIn: "/login",
     error: "/error",
   },
-  secret: process.env.AUTH_SECRET,
+  secret:
+    process.env.AUTH_SECRET || "IBq7Jk+ZYYvcxMWQQfsDCTC3Q0qYO+ukk0wyYRKuSqk=",
```

Comment:
Why are you hardcoding an auth secret?

Reply:
i didn't do anything around auth, probably from my last pull, but would fixx it sir